### ISSUE TITLE
Group audio and video (multiple audio)

### ISSUE THREAD

**Issue by DajMiliona** [rowrawer/stream-detector#115]
It is a possible slect audio to download with mp4?
Example https://www.europarl.europa.eu/plenary/pl/vod.html?mode=chapter&vodLanguage=PL&vodId=06fba284-7572-0ce6-22f8-200467512d63&date=20220518#
In VLC we can select language audio and play and record. It is a way to fast download video witch selected audio?

**Comment by rowrawer** [comment]
No, the addon doesn't perform any invasive procedures aside from detecting the entries. It doesn't download and read the files for its own use in any way so it doesn't know what they identify as. You have to pick and choose the desired files yourself.
